web_external/ImagesGallery/Listing/ImagesPagingPane.js

```diff
@@ -53,6 +53,12 @@ isic.views.ImagesPagingPane = Backbone.View.extend({
             staticImageRoot: girder.staticRoot + '/built/plugins/isic_archive/extra/img/'
         }));
 
+        // Tooltips for buttons inside a "btn-group" must be attached to another element, or else
+        // the "btn-group" size will be disrupted when they appear
+        this.$('.btn-group>[data-toggle="tooltip"]').tooltip({
+            trigger: 'hover',
+            container: this.$el
+        });
```

Comment:
Won't this call `.tooltip()` twice on at least some of the tooltip elements, since we already call it below? Perhaps that's harmless, but certainly unnecessary. Can the call below just be removed?